lidarr/readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## Notice
 
-Please note that use of arr-scripts modified Lidarr's behavior and is not supported by the Lidarr community.
+Please note that the use of arr-scripts modifies Lidarr's behavior and is not supported by the Lidarr community.
```

Comment:
Maybe change it to:

"Please note that use of arr-scripts is not supported by the Lidarr community."

Reply:
This is perfect.